manifests/g/GuinpinSoft/MakeMKV/v1.17.2/GuinpinSoft.MakeMKV.installer.yaml

```
@@ -0,0 +1,12 @@
+# Automatically updated by the winget bot at 2022/Oct/13
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GuinpinSoft.MakeMKV
+PackageVersion: v1.17.2
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://www.makemkv.com/download/Setup_MakeMKV_v1.17.1.exe

```

```suggestion
  InstallerUrl: https://www.makemkv.com/download/Setup_MakeMKV_v1.17.1.exe
```
InstallerUrl needs to be fixed.